### ISSUE TITLE
[Wii] PC-Engine CD games do not work

### ISSUE THREAD

**Issue by gingerbeardman** [libretro/RetroArch#2173]
Just a black screen.

**Comment by Kivutar** [comment]
You do have the bios, right?

**Comment by gingerbeardman** [comment]
Yes, but thanks for checking! 

More reports: http://libretro.com/forums/showthread.php?t=3690

**Comment by fr500** [comment]
please report this in the mednanfen pc engine libretro repository

**Comment by gingerbeardman** [comment]
A link would have been helpful.

**Comment by Alcaro** [comment]
tl;dr [here](https://github.com/libretro/beetle-pce-fast-libretro)

Normally I'd whine about laziness from both of you, but in this case, the correct repo is rather hard to find - neither searching for 'medna' nor 'engine' leads anywhere sensible.

Searching for 'pc' gives me four answers: [pcsx_rearmed](https://github.com/libretro/pcsx_rearmed), [beetle-pce-fast-libretro](https://github.com/libretro/beetle-pce-fast-libretro), [beetle-pcfx-libretro](https://github.com/libretro/beetle-pcfx-libretro) and [pcsx1-libretro](https://github.com/libretro/pcsx1-libretro). A quick look suggests three of them are for PlayStation; the PCSXes say it in the description, and pce-fast's readme identifies as Mednafen PSX libretro. (Our Mednas are named Beetle due to author request.)

It would then not be unreasonable for someone unfamiliar with '90s gaming consoles to assume that PC-FX is a PC-Engine emulator; many other emulators, such as Dolphin, Citra, Xenia and higan have completely nondescriptive names (at least for people not familiar with the console), so a name containing "PC" doesn't raise any eyebrows.

Digging into the source code [reveals that](https://github.com/libretro/beetle-pce-fast-libretro/blob/master/libretro.cpp#L846) pce-fast is the correct one, and its readme is incorrect.

And I can't find any non-Fast medna-PCE, but with all the weird repo names and unhelpful descriptions running around here, I can't be sure about anything.

Confusing stuff.

**Comment by gingerbeardman** [comment]
Thanks for your help :) filing there now

**Comment by gingerbeardman** [comment]
Done: https://github.com/libretro/beetle-pce-fast-libretro/issues/16
